02LessonTwo/homework/index.html

```diff
@@ -1,26 +1,81 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>First Site</title>
+  <title>Resume - Violeta Campos</title>
   <link rel="stylesheet" href="css/style.css" type="text/css">
 </head>
 <body>
 
-    <h1>Goodbye, space!</h1>
+  <section>
 
-    <div>
-      <img src="http://cdn.theatlantic.com/assets/media/img/photo/2015/11/images-from-the-2016-sony-world-pho/s01_130921474920553591/main_900.jpg?1448476701" alt="Jumping Hamster"
-      title="Jumping Hamster">
+    <div class="name">
 
-      <p>Who doesn't love a jumping hamster?</p>
+      <h1>Violeta Campos</h1>
+
+      <h5>10800 Lakeline Blvd, Austin TX</h5>
 
-      <a href="https://slack.com/" target="_blank">Go to Slack!</a>
     </div>
 
-  <!-- <div></div>
-  <p></p>
-  <img>
-  <a></a> -->
+  </section>
+
+  <div class="text">
+
+    <h2>Bio</h2>
+
+      <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+
+  </div>
+
+  <div class="text">
+
+  <h3>Hobbies</h3>
+
+    <ul>
+
+      <li>Movies at the Drafthouse</li>
+      <li>Biking on different trails around Austin</li>
+      <li>Trying a new restaurant</li>
+
+    </ul>
+
+  <h3>Things I Don't Like</h3>
+
+    <ul>
+
+      <li>Traffic</li>
+      <li>Bad Coffee</li>
+      <li>Working Out</li>
+
+    </ul>
+
+  </div>
+
+<div id="pictures">
+
+  <h3>TV Shows I Watch</h3>
+
+  <h4>Games of Thrones</h4>
+
+    <a href="http://www.hbo.com/game-of-thrones"><img src="http://i.lv3.hbo.com/assets/images/series/game-of-thrones/home/season-06/160223-s6-key-art-1920x700.jpg" alt="Faces" title="Game of Thrones" style="height:300px; width:500px"/>
+    </a>
+
+  <h4>Homeland</h4>
+
+    <a href="http://www.sho.com/sho/homeland/home"><img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTrQ_PYtS99YNHqv9hGBiEL3IjmCcbrjP_WcZj4CWCuP2XmYq4v7g" alt="Claire Danes, Damien Lewis" title="Homeland" style="height:300px; width:400px"/>
+    </a>
+
+  <h4>Breaking Bad</h4>
+
+    <a href="http://www.amc.com/shows/breaking-bad"><img src="http://unspoiledpodcasts.com/wp-content/uploads/2014/09/Breaking-Bad.jpg" alt="Bryan Cranston, Aaron Paul" title="Breaking Bad" style="height:300px; width:400px"/>
+    </a>
+
+</div>
+
```

Comment:
a great start, Violet! clean, organized code that is almost 100% correctly nested. Remember that all of your content inside the body is nested and is thus in need of a base indentation.